CHANGELOG.md

```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file.
 <!-- ignore lint rules that are often triggered by content generated from commits / git-cliff -->
 <!-- markdownlint-disable line-length no-bare-urls ul-style emphasis-style -->
 
+## ratatui-termion - [0.1.0-alpha.2](https://github.com/ratatui/ratatui/compare/ratatui-termion-v0.1.0-alpha.1...ratatui-termion-v0.1.0-alpha.2) - 2025-02-11
+
+### Miscellaneous Tasks
+
+- [0000000](https://github.com/ratatui/ratatui/commit/0000000) *(uncategorized)* Updated the following local packages: ratatui-core
```

Comment:
`000000` ? 👀

Reply:
looks like a bug - that commit ain't a thing

Reply:
It's a commit that release-plz adds to say that a dependency was updated. You csn probably filter it in git cliff config